### ISSUE TITLE
+More link (event limit) for timeGrid

### ISSUE THREAD

**Issue by therohin** [fullcalendar/fullcalendar#4218]
Having the option to enable +more link (event limit) for agendaDay and agendaWeek views.
There are scenarios where there might be multiple events happening at the same time and when events go above 3 or 4, it looks really bad in the UI.
Having an option to set an event limit in the agenda views becomes very important in such scenarios.

Attached a view with multiple events created by an automated system in our application and a suggested view/UI for + more link.
**AgendaWeek:**
![image](https://user-images.githubusercontent.com/24406128/41523317-2cd8ef98-72f7-11e8-9362-279197a5b0dd.png)
**AgendaDay**
![image](https://user-images.githubusercontent.com/24406128/41523366-6048df0a-72f7-11e8-8d7f-d82f10adfb00.png)
**Suggested UI**
![image](https://user-images.githubusercontent.com/24406128/41523602-3dbef66c-72f8-11e8-9800-2cce685ea8ad.png)

**Comment by acerix** [comment]
Thanks for the suggestion, I made this demo showing the issue:

https://codepen.io/anon/pen/ERQWer?&editors=001

So, you are looking for a setting like "eventLimit" but that would set the maximum number of events that will show next to eachother in agenda view?

**Comment by therohin** [comment]
Yes, the demo shows the issue perfectly. So to limit the number of events shown in agenda views, especially in agendaDay, so the events are visible.
Also I feel this might also be an issue while showing events in vertical resource view with Scheduler.

**Comment by therohin** [comment]
As the mentioned in the documentation, the setting is only applicable for "all-day" rows in agenda days and not for time based rows/cells.

**Comment by cellis-voyager** [comment]
Support

**Comment by kumarrk5** [comment]
I also indeed of this issue solution.

**Comment by rishu2482** [comment]
I also in-need of this issue solution.
Somebody please fix this.

**Comment by guilhermefalves** [comment]
Any news?

**Comment by bipinm25** [comment]
Please support i also want this feature

**Comment by Anyoks** [comment]
Also met the same issue.

**Comment by m4xpwr** [comment]
I have the same issue.  Would love a solution.

**Comment by sudarshanrai89** [comment]
Is this feature targeted for any release?

**Comment by badrshs** [comment]
any update please ? i have same problem

**Comment by Pinaksh** [comment]
Event Limit will help us actually with lot of events UI becomes bad. Please consider it in next releases. Thanks.

**Comment by arturogsaizapw** [comment]
any update ?  (=

**Comment by timsar2** [comment]
Confirm,
eventLimit not working for timeGrid.

```
eventLimit = true;
views: {
    agenda: {
      eventLimit: 3
    },
    timeGrid: {
      eventLimit: 3
    }
  }
```

**Comment by Yokomoko** [comment]
Any workaround for this?

**Comment by lao9s** [comment]
Is possible to display always +1,+2..etc? I don't want to display even one event on day cell.

**Comment by tsyrya** [comment]
@acerix that would be awesome if you add that feature to the v5 release.

**Comment by ehsantekgenisys** [comment]
@acerix @arshaw  Is that feature added in v5 release?
or any other way to resolve this issue?

**Comment by lgdal** [comment]
@acerix @arshaw Any update on this? or even a suggestion of a workaround for now?

**Comment by ronaldosjr** [comment]
Any news about this, guys?

**Comment by tibinvpaul** [comment]
Would be great if you can release this feature. The UI is cluttered for users with many events! Thank you.

**Comment by ngthtung** [comment]
+1, dayMaxEventRows not work at timeGridWeek in version 5

**Comment by Tiagojp0799** [comment]
Any news about this?

**Comment by arshaw** [comment]
This has been released in v5.7.0!
blog post: https://fullcalendar.io/blog/2021/05/new-features-in-v570

The `eventMaxStack` setting.

example: https://codepen.io/arshaw/pen/rNyxNLL?editors=0010
